ReactNative/Views/LogoView.swift

```diff
@@ -9,10 +9,42 @@
 import Foundation
 import React
 
+/*
+ * Remeber to assign url and set view constraints.
+ */
 class LogoView: UIView {
-    private var url: String?
-    private lazy var reactView: UIView = {
-        guard let url = self.url else { return UIView() }
+    public var url: String? {
+        didSet {
+            // To prepareForReuse set url to nil - this will remove React view
+            if url == nil {
+                guard let reactView = self.reactView else { return }
+                reactView.removeFromSuperview()
+                self.reactView = nil
+            }
+        }
+    }
+
+    private var reactView: RCTRootView?
+    private var size: CGFloat?
```

Comment:
seeing property size, I expect to have a type of CGSize, something which descrbes at least 2 dimensional size of the object. I would rename it preferredHeight or something like this